src/Item/Item.model.ts

```diff
@@ -16,7 +16,7 @@ export class Item extends Model<ItemAttributes> {
 
   static findDBApprovalDataByCollectionId(collectionId: string) {
     return this.query<DBItemApprovalData>(SQL`
-      SELECT items.id, items.urn_suffix, item_curations.local_content_hash
+      SELECT items.id, item_curations.local_content_hash
```

Comment:
```suggestion
      SELECT items.id, item_curations.content_hash
```
I believe the column is called `content_hash`